### ISSUE TITLE
download fails for virtualenv.py

### ISSUE THREAD

**Issue by Sriram21** [pypa/virtualenv#1837]
Hi,

File not found : https://raw.githubusercontent.com/pypa/virtualenv/master/virtualenv.py

I have been using the above link to download virtual env.

This has been throwing error : 404: Not Found

Could you please let us know why this was removed?

Regards 
Sriram

**Comment by gaborbernat** [comment]
``virtualenv.py`` is no longer a thing with release ``20``. Prefer installation methods described at https://virtualenv.pypa.io/en/latest/installation.html#installation.

**Comment by Sriram21** [comment]
Hi

Is it possible to provide me a backup of that file.

I need to immediately build a docker image and make use of it in production

Regards
Sriram

**Comment by gaborbernat** [comment]
That file is deprecated so use at your own peril, however, you can still find it under https://github.com/pypa/virtualenv/blob/legacy/virtualenv.py. Consider though moving to one of the better installations from above, e.g. the zipapp is a much better way.

**Comment by Sriram21** [comment]
Hi @gaborbernat 

I tried the zipapp as you suggested. I ran into two issues.

When i ran python3.3 virtualenv.pyz it throws ImportError: No module named 'pathlib' error

When i ran python3.6 virtualenv.pyz it throws a different error.

![zipapp](https://user-images.githubusercontent.com/6112284/82446968-3401f600-9ac5-11ea-8708-0ee3cd7d2bd7.jpg)

How can i make this thing to work.. Any help is much appreciated.

Regards,
Sriram
